README.md

```diff
@@ -3,14 +3,15 @@
 Grammar for AWS Event Rules:
 <https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-event-patterns-content-based-filtering.html>
 
-Event Rules are JSON documents that are used to filter other JSON documents.  
-On AWS EventBridge, they are used to filter AWS CloudWatch Events.  
+Event Rules are JSON documents that are used to filter other JSON documents.
```

Comment:
there were double-space line breaks at the end of these lines that were accidentally removed.